Singularity/Singularity/Platform/Mine.cs

```diff
@@ -17,9 +17,9 @@ class Mine : PlatformBlank
 
         public Mine(Vector2 position, Texture2D platformSpriteSheet, Texture2D baseSprite, ResourceMap resource): base(position, platformSpriteSheet, baseSprite, new Vector2(position.X + PlatformWidth / 2f, position.Y + PlatformHeight - 36))
         {
+            //Add possible Actions in this array
             mIPlatformActions = new IPlatformAction[2];
```

Comment:
Why is the array size 2? Is it limited to 2? There is no action in [1], right? (Potential NullPointer)

Reply:
Its from an earlier time when I didnt knew that other actions can be on the platform too (like blueprints from others). I will change it.